lib/seattleflu/db/cli/command/reportable_conditions.py

```diff
@@ -166,6 +170,13 @@ def send_slack_post_request(record: NamedTupleCursor.Record, url: str) -> reques
         "condition": record.lineage
     }
 
+    if not record.site:
+        additional_data = {
```

Comment:
Do we want a higher level key like `manifest` to make it clear this is manifest data?

Reply:
I think that's a good idea.